html-css/week1/nadiia/order-desktop/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Payment page</title>
+    <!--Netlify deployment: https://mellifluous-blini-c03940.netlify.app-->
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="h1">Order summary</h1>
+      <p class="p Total">Total<span>$333/month</span></p>
+      <hr class="hr" />
+      <p class="p Total_2">Total<span>$100</span></p>
```

Comment:
```suggestion
      <p class="p Total">Total<span>$333/month</span></p>
      <p class="p Total_2">Total<span>$100</span></p>
```

I'd still suggest removing the `<hr />` tag and adding `border-top: 2px solid white;` on `.Total_2`.